google-cloud-env/lib/google/cloud/env.rb

```diff
@@ -353,7 +351,7 @@ def metadata?
           rescue *METADATA_FAILURE_EXCEPTIONS
             retries_remaining -= 1
             if retries_remaining >= 0
-              sleep @retry_interval if @retry_interval
+              sleep @retry_interval
```

Comment:
A simple delay and retry is sufficent? An incremental backoff is not required?

Reply:
I was going to say I didn't think it was important... but I guess it's not hard to put in.